etsin_finder/frontend/js/stores/view/qvain.js

```diff
@@ -803,6 +841,14 @@ class Qvain {
     // Keywords
     this.keywordsArray = researchDataset.keyword || []
 
+    // SubjectHeadings/Themes
```

Comment:
clear the subjectHeadingsArray here to make sure it has no leftovers from other datasets